source/time_tables/tests/departure_board_test_data.h

```diff
@@ -26,7 +26,9 @@ struct calendar_fixture {
         b.vj("line:A", "10101", "", true, "weekend")("stop1", 20 * 3600, 20 * 3600 + 10 * 60)("stop2", 21 * 3600, 21 * 3600 + 10 * 60);
 
         // and one everytime
-        b.vj("line:A", "1100101", "", true, "all")("stop1", 15 * 3600, 15 * 3600 + 10 * 60)("stop2", 16 * 3600, 16 * 3600 + 10 * 60);
+        auto& builder_vj = b.vj("line:A", "1100101", "", true, "all")("stop1", 15 * 3600, 15 * 3600 + 10 * 60)("stop2", 16 * 3600, 16 * 3600 + 10 * 60);
```

Comment:
line length too (was previously :sob: ) you can use `("stop1", "15:00"_t, "15:10"_t)("stop2", "16:00"_t, "16:10"_t);`